diff --git a/semitone/build.gradle b/semitone/build.gradle
index a9c9dcf..4db0a16 100644
--- a/semitone/build.gradle
+++ b/semitone/build.gradle
@@ -15,7 +15,8 @@ buildscript {
     }
 }
 
-apply plugin: 'com.android.application'
+// apply plugin: 'com.android.application'
+apply plugin: 'com.android.library'
 
 repositories {
     mavenCentral()
@@ -37,10 +38,11 @@ android {
 
     lintOptions {
         disable "GoogleAppIndexingWarning"
+        disable "AppCompatCustomView"
     }
 
     defaultConfig {
-        applicationId "mn.tck.semitone"
+//        applicationId "mn.tck.semitone"
         minSdkVersion 16
         targetSdkVersion 28
         versionCode 3
